static/index.html

```diff
@@ -779,20 +775,20 @@ <h3 class="type-5 text-lagunita-light">Stone</h3>
             </div>
           </article>
 
-          <article id="color-palette-foggy" class="rs-mb-5">
+          <article id="color-palette-fog" class="rs-mb-5">
             <header>
-              <h3 class="type-5 text-lagunita-light">Foggy</h3>
+              <h3 class="type-5 text-lagunita-light">Fog</h3>
             </header>
             <div class="grid md:grid-cols-2 lg:grid-cols-3 gap-xl md:gap-lg 2xl:gap-2xl">
               <div
-                class="rs-py-4 rs-px-1 flex justify-center content-center font-semibold border border-black-10 text-black bg-foggy">
+                class="rs-py-4 rs-px-1 flex justify-center content-center font-semibold border border-black-10 text-black bg-fog">
```

Comment:
For a while we've added the alias `foggy` to the color `fog` since in the original color palette (decanter v5 or before), fog was a different color compared to our current fog. I think we're ok now. I'm leaving the `foggy` alias in, but we're updating our code to use the OG `fog` whenever possible.